.circleci/config.yml

```diff
@@ -250,10 +250,10 @@ jobs:
           working_directory: packages/server-wallet
           command: |
             sudo apt install -y postgresql-client || true
-            createdb ping $PSQL_ARGS
-            SERVER_DB_NAME=ping yarn db:migrate
-            createdb pong $PSQL_ARGS
-            SERVER_DB_NAME=pong yarn db:migrate
+            createdb payer $PSQL_ARGS
```

Comment:
Outside the scope of this PR, but this seems like a bad name for a database. It's plausible that I might already have a database called `payer`. I think it's sufficiently unlikely that I have one named `sc_e2e_payer`.